ts/parts/AreaSeries.ts

```diff
@@ -531,11 +531,20 @@ seriesType<Highcharts.AreaSeries>(
 
             topPath = getGraphPath.call(this, graphPoints, true, true);
             if (series.chart.is3d()) {
+                var options3d = (series as any).chart.options.chart.options3d;
```

Comment:
Do you have an idea how to remove this `any` cast? We have a case where we are in 3d graph so `options3d` are defined, however, without this casting I would need to make unnecessary check in `if` statement